ddl/column.go

```diff
@@ -574,6 +584,30 @@ func modifyColumnFromNull2NotNull(w *worker, t *meta.Meta, dbInfo *model.DBInfo,
 	return nil
 }
 
+// modifyColsFromNull2NotNull modifies the type definitions of 'null' to 'not null'.
+// Introduce the `mysql.PreventNullInsertFlag` flag to prevent users from inserting or updating null values.
+func modifyColsFromNull2NotNull(w *worker, dbInfo *model.DBInfo, tblInfo *model.TableInfo, cols []*model.ColumnInfo) error {
```

Comment:
I think `modifyColsFromNull2NotNull` and `modifyColumnFromNull2NotNull` are nearly the same. But `modifyColsFromNull2NotNull` doesn't set the job state. There are some duplicated work here.